Submodels/Parameterizations.py

```
@@ -0,0 +1,775 @@
+- V1.0: Adapted version of a parameterization using only the bezier coefficients to give an improved fit to the reference data. 
+
+"""
+
+
+
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy import interpolate, optimize
+from typing import Tuple
+
+class AirfoilParameterization():
```

Ruff (0.8.2)

Unnecessary parentheses after class definition

Remove parentheses

(UP039)